sqlglot/dialects/sqlite.py

```diff
@@ -94,9 +90,17 @@ class Generator(generator.Generator):
             exp.TimeStrToTime: lambda self, e: self.sql(e, "this"),
             exp.TryCast: no_trycast_sql,
             exp.GroupConcat: _group_concat_sql,
-            exp.Fetch: _fetch_sql,
         }
 
+        def fetch_sql(self, expression):
+            return self.limit_sql(exp.Limit(expression=expression.args.get("count")))
+
+        def least_sql(self, expression):
+            if len(expression.expressions) > 1:
+                return rename_func("MIN")(self, expression)
+
+            return self.expressions(expression)
```

Comment:
`MIN` with one argument in sqlite is interpreted as an aggregate function instead of a scalar one, which is why I took separate cases here.